docs/source/index.rst

```diff
@@ -31,13 +31,13 @@ architectures, and common image transformations for computer vision.
    :maxdepth: 2
    :caption: Package Reference
 
-   datasets
    transforms
    models
-   feature_extraction
+   datasets
+   utils
    ops
    io
-   utils
+   feature_extraction
```

Comment:
Not clubbing with models? Potentially this is very new feature (and hence low popularity), but the docs are best read after reading models.

Reply:
We'll likely merge it into the models page once we get to the more general revamp of the models page